business-registry-dashboard/app/locales/en-CA.ts

```diff
@@ -161,6 +161,18 @@ export default {
         title: 'Business Unavailable',
         description: 'You are not authorized to access the business you wish to {action}. Please add this business to your table to continue.'
       }
+    },
+    magicLinkUnauthorized: {
+      title: 'Unable to Manage Business',
+      description: 'The account that requested authorisation does not match your current account. Please log in as the account that initiated the request.'
```

Comment:
1. fix "authorization"
2. "log in as" vs "log in to" or "log into" or "login to"?

Reply:
Good points Sev. From the UXPin: 
![image](https://github.com/user-attachments/assets/94243ebf-01f8-4147-978d-d85a93d843a2)

REF: https://preview.uxpin.com/c0e1953a537febcbb94df1856812c0ea4b52ccb3#/pages/161898947/simulate/sitemap?mode=i

Reply:
OK. You may resolve this conversation. (I do not have permission.)